features/cloud_pro_clone.feature

```diff
@@ -18,8 +18,8 @@ Feature: Creating golden images based on Cloud Ubuntu Pro instances
     And I run `pro status --format yaml` with sudo
     Then stdout matches regexp:
       """
-      name: fips-updates
-      status: enabled
+      \s*name: fips-updates
+      \s*status: enabled
```

Comment:
Hi @lucasmoura , I noticed the use of `\s*` in this regexp matching. Could you help me understand why it's used here and not in other similar situations when matching stdout with regexp?

Reply:
This is just because we are testing the yaml output and this output is formatted in a way that the extra spaces are required here.

The problem is that the gherkin formatter is removing those spaces from the file and that's why I have used the regex.

Reply:
Ah yes, the gherkin formatter is very picky about extra lines or spaces in output comparisons. Thanks!